lib/hammer_cli_foreman_discovery/discovery_rule.rb

```diff
@@ -51,15 +52,13 @@ class CreateCommand < HammerCLIForeman::CreateCommand
       failure_message _("Could not create the rule")
 
       include HammerCLIForemanDiscovery::CommonDiscoveryRuleUpdateOptions
-      build_options
```

Comment:
removed this line since this rewrites CommonDiscoveryRuleUpdateOptions functionality which is quite weird.